packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -332,6 +332,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     markNeedsTextLayout();
   }
 
+  /// Character used to obscure text if [obscureText] is true.
```

Comment:
My english is not perfect. I have been wanting to use `used to` in this kind of sentence. Can they actually be used in this way? Otheriwse, I will suggest `Character used for obscuring text if [obscureText] is true.`

Reply:
I say yes!  These two sentences are both correct and mean the same thing:

"A shovel is a tool used to dig holes."
"A shovel is a tool used for digging holes."

That said, maybe your way is more clear.  My way could be confused for something like this:

"My shovel used to dig holes, but now it doesn't anymore."